locales/fr.json

```diff
@@ -74,6 +74,7 @@
   },
   "collections": {
     "general": {
+      "all_of_collection": "Tous les {{ collection }}",
```

Comment:
We actually got rid of this in our Shopify themes. There are 3 issues with passing the collection title:

* Grammatical. This will work half the time in French for example, because words have gender there ( same in Spanish ! ). `Tous les` only works with masculine words.
* You often end up having a collection which name is 'All Products'. Then you have 'All all products'.
* To override the automatic /collections/all collection, merchants need to create a collection with handle `all` and when they do that, they often create a collection with title **is** `All` ( to end up with the right handle ) and then you will see `All all`.

On paper this seems like the best idea though! It's just too bad we cannot really make it work. :'(